.tool-versions

```diff
@@ -1,5 +0,0 @@
-# If updating this, you *must* also change the channel in rust-toolchain.toml
-# renovate-automation: rustc version
-rust 1.61.0
```

Comment:
@o0Ignition0o Curious — this PR was intended to remove the dependency on Node.js, which was indeed great, but I'm curious why you chose to remove `rust` from `.tool-versions` as well?  Would we consider adding it back?  Do we think it's unnecessary?

Reply:
As far as I know `.tool-versions` is only used by ASDF. In my experience folks writing Rust regularly have `$(which cargo)` pointing to rustup, not ASDF. With this PR, ASDF is not needed at all to contribute to the Router.